drivers/temperature/adt7420/adt7420.h

```diff
@@ -42,92 +42,151 @@
 /******************************************************************************/
 /***************************** Include Files **********************************/
 /******************************************************************************/
-#include <stdint.h>
+
+#include "spi.h"
 #include "i2c.h"
 
 /******************************************************************************/
 /************************** ADT7420 Definitions *******************************/
 /******************************************************************************/
 
 /* ADT7420 address */
-#define ADT7420_A0_PIN(x)	(((x) & 0x1) << 0) // I2C Serial Bus Address Selection Pin
-#define ADT7420_A1_PIN(x)	(((x) & 0x1) << 1) // I2C Serial Bus Address Selection Pin
-#define ADT7420_ADDRESS(x,y)	(0x48 + ADT7420_A1_PIN(x) + ADT7420_A0_PIN(y))
-
-/* ADT7420 registers */
-#define ADT7420_REG_TEMP_MSB		0x00 // Temperature value MSB
-#define ADT7420_REG_TEMP_LSB		0x01 // Temperature value LSB
-#define ADT7420_REG_STATUS		0x02 // Status
-#define ADT7420_REG_CONFIG		0x03 // Configuration
-#define ADT7420_REG_T_HIGH_MSB		0x04 // Temperature HIGH setpoint MSB
-#define ADT7420_REG_T_HIGH_LSB		0x05 // Temperature HIGH setpoint LSB
-#define ADT7420_REG_T_LOW_MSB		0x06 // Temperature LOW setpoint MSB
-#define ADT7420_REG_T_LOW_LSB		0x07 // Temperature LOW setpoint LSB
-#define ADT7420_REG_T_CRIT_MSB		0x08 // Temperature CRIT setpoint MSB
-#define ADT7420_REG_T_CRIT_LSB		0x09 // Temperature CRIT setpoint LSB
-#define ADT7420_REG_HIST		0x0A // Temperature HYST setpoint
-#define ADT7420_REG_ID			0x0B // ID
-#define ADT7420_REG_RESET		0x2F // Software reset
+#define ADT7420_A0_PIN(x)	            (((x) & 0x1) << 0) // I2C Serial Bus Address Selection Pin
```

Comment:
Please don't change the spacing is really hard to see the changes. 
Anyway the indentation is not ok, use tab with a width of 8 space

Reply:
Spacing has been restored as in the original file.